repose-aggregator/src/docs/asciidoc/_backends/haml/html5/document.html.haml

```diff
@@ -0,0 +1,122 @@
+    %script(src="#{attr :prettifydir, 'http://cdnjs.cloudflare.com/ajax/libs/prettify/r298'}/prettify.min.js")
+    %script document.addEventListener('DOMContentLoaded', prettyPrint)
+  - if attr? 'stem'
+    %script(type='text/x-mathjax-config')
+      :plain
+          MathJax.Hub.Config({
+            tex2jax: {
+              inlineMath: [['\\(','\\)']],
+              displayMath: [['\\[','\\]']],
+              ignoreClass: 'nostem|nolatexmath'
+            },
+            asciimath2jax: {
+              delimiters: [['\\$','\\$']],
+              ignoreClass: 'nostem|noasciimath'
+            }
+          });
+    %script(type='text/javascript' src='http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-MML-AM_HTMLorMML')
+  - unless (docinfo_content = docinfo).empty?
+    =docinfo_content
+  %body{:id => @id, :class=>[doctype, ((attr? 'toc-class') && (attr? :toc) && (attr? 'toc-placement', 'auto') ? "#{attr 'toc-class'} toc-#{attr 'toc-position', 'left'}" : nil)], :style=>("max-width: #{attr 'max-width'};" if (attr? 'max-width'))}
+    - unless noheader
+      -# AsciiDoc leaves an empty header div even if there's no doctitle
+#header
+  - if has_header?
+    - unless notitle
+      %h1=@header.title
+    - if (attr? :author) || (attr? :revnumber) || (attr? :revdate) || (attr? :revremark)
+      .details
+        - if attr? :author
+          %span#author.author=attr :author
+          %br
+          - if attr? :email
+            %span#email.email=sub_macros(attr :email)
+            %br
+          - if (authorcount = (attr :authorcount).to_i) > 1
+            - (2..authorcount).each do |idx|
+              %span{:id=>"author#{idx}", :class=>"author"}=(attr "author_#{idx}")
+              %br
+              - if attr? "email_#{idx}"
+                %span{:id=>"email#{idx}", :class=>"email"}= sub_macros(attr "email_#{idx}")
+        - if attr? :revnumber
+          %span#revnumber #{((attr 'version-label') || '').downcase} #{attr :revnumber}#{',' if (attr? :revdate)}
+        - if attr? :revdate
+          %span#revdate=attr :revdate
+        - if attr? :revremark
+          %br
+          %span#revremark=attr :revremark
+  - if (attr? :toc) && (attr? 'toc-placement', 'auto')
+    #toc{:class=>(attr 'toc-class', 'toc')}
+      #toctitle=attr 'toc-title'
+      =converter.convert self, 'outline'
+  #content
+    =content
+  - unless !footnotes? || attr?(:nofootnotes)
+    #footnotes
+      %hr
+      - footnotes.each do |fn|
+        .footnote{:id=>['_footnote', fn.index]}
+          = succeed ". #{fn.text}" do
+            %a(href="#_footnoteref_#{fn.index}")=fn.index
+  - unless nofooter
+    #footer
+      #footer-text
+        - if attr? 'project-version'
+          = precede %(#{attr 'version-label'} #{attr 'project-version'}) do
+            - if (attr? :revnumber) || (attr? 'last-update-label')
+              %br
```

Comment:
The four lines above this comment are what I added to the default template.

I should note that I'm re-using the `version-label` attribute, which will only be a problem if, in the future, we want to provide a `revnumber` and label it differently than we label the project version.